syntaxes/templ.tmLanguage.json

```diff
@@ -389,65 +386,63 @@
         }
       ]
     },
-    "empty-import-expression":{
-      "match": "@(?:[A-z_][A-z_0-9]*\\.)?[A-z_][A-z_0-9]*\\(.*\\)\\s*$",
-      "captures": {
-        "0": {
-          "patterns": [
-            {
-              "include": "#import-expression-start"
-            }
-          ]
-        }
-      }
-    },
     "import-expression": {
-      "begin": "@(?:[A-z_][A-z_0-9]*\\.)?[A-z_][A-z_0-9]*\\(.*\\)\\s*{$",
-      "beginCaptures": {
-        "0": {
+      "patterns": [
+        {
+          "name": "import-expression.templ",
```

Comment:
This outermost pattern matches from `@` to the first opening parenthesis and provides the `keyword.control.go` to the `@` and `source.go` to the function name/opening parenthesis.